### ISSUE TITLE
About the dataset

### ISSUE THREAD

**Issue by sherlhw** [IDEA-Research/Motion-X#34]
Thank you very much for your work, but I have some questions. How do I download IDEA400 dataset? Does IDEA400 dataset have corresponding pictures or videos data? And when will the videos data of the Motion-X  be released?

**Comment by ffli** [comment]
In the readme, the IDEA 400 dataset download link

**Comment by sherlhw** [comment]
![img_v3_026b_94f10de2-d8e4-4494-b924-c73a7b1c620g](https://github.com/IDEA-Research/Motion-X/assets/97173205/5bea1e35-25d5-4b21-bebe-f7426e99c120)
Is this the link?

**Comment by linjing7** [comment]
Yes, please fill in the register table and we will send you a downloading link.
